src/typinf.c

```diff
@@ -376,7 +379,10 @@ class TypeInfoDtVisitor : public Visitor
         assert(name);
         size_t namelen = strlen(name);
         dtsize_t(pdt, namelen);
-        dtabytes(pdt, 0, namelen + 1, name);
+        dtxoff(pdt, d->csym, Type::typeinfofunction->structsize);
```

Comment:
I plan to do that later, but it will require a corresponding binary change to druntime.